src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -103,21 +103,21 @@ def load_arguments(self, _):
 
     # Identity
     with self.argument_context('backup vault identity assign') as c:
-        c.argument('vault_name', vault_name_type)
+        c.argument('vault_name', vault_name_type, options_list=['--name', '-n'], id_part='name')
```

Comment:
Could you keep the original parameter name and then add the new parameter names? This can avoid breaking change to users
```suggestion
        c.argument('vault_name', vault_name_type, options_list=['--vault-name', '--name', '-n'], id_part='name')
```

Reply:
Yeah i can do it. Actually this command is not released yet. It was added after the last release so i thought it would be better changing it before the next release.

Reply:
Do i need to change it? the command is not released yet so it's not a breaking change.

Reply:
Without breaking change, we don't need to modify it. Just keep it~